modules/restore-replace-stopped-baremetal-etcd-member.adoc

```diff
@@ -167,7 +167,7 @@ secret "etcd-serving-openshift-control-plane-2" deleted
 . Delete the control plane machine.
 +
 If you are running installer-provisioned infrastructure, or you used the Machine API to create your machines, follow these steps. Otherwise, you must create the new control plane node using the same method that was used to originally create it.
-
+//todo: assess for CPMS impact
```

Comment:
Backup and restore > Control plane backup and restore > Replacing an unhealthy etcd member > [Replacing an unhealthy bare metal etcd member whose machine is not running or whose node is not ready](https://53397--docspreview.netlify.app/openshift-enterprise/latest/backup_and_restore/control_plane_backup_and_restore/replacing-unhealthy-etcd-member.html#restore-replace-stopped-baremetal-etcd-member_replacing-unhealthy-etcd-member)

Reply:
I am guessing this is unchanged because it's for bare metal?

Reply:
Yes, on baremetal CPMS doesn't make a lot of sense, so I'd leave this alone